### DISCUSSION TITLE
Pathway score calculations?

### DISCUSSION THREAD

**Discussion by junli1988** [egeulgen/pathfindR#143] [Q&A] [answered]
Hi, I have a new question. I've read through your article on Frontiers in Genetics and I'm interested in the pathway scores function of the package. So when I'm using the command for score_terms(), should I use normalized data or raw transcript counts in the exp_matrix  line? Does it matter? Thanks in advance.

**Comment by egeulgen** [comment, marked as answer]
Hello,
I would suggest using normalized data.
-E
